src/s1reader/utils/plot_bursts.py

```diff
@@ -0,0 +1,140 @@
+                 'border':[]}
+    i_subswath = [1, 2, 3]
+    pol = 'vv'
+    orbit_list = glob.glob(f'{orbit_dir}/*EOF')
+    orbit_path = get_orbit_file_from_list(slc, orbit_list)
+
+    for subswath in i_subswath:
+        ref_bursts = load_bursts(slc, orbit_path, subswath, pol)
+        for burst in ref_bursts:
+            burst_map['burst_id'].append(burst.burst_id)
+            burst_map['length'].append(burst.shape[0])
+            burst_map['width'].append(burst.shape[1])
+            burst_map['spacing_x'].append(x_spacing)
+            burst_map['spacing_y'].append(y_spacing)
+            burst_map['first_valid_line'].append(burst.first_valid_line)
+            burst_map['last_valid_line'].append(burst.last_valid_line)
+            burst_map['first_valid_sample'].append(burst.first_valid_sample)
+            burst_map['last_valid_sample'].append(burst.last_valid_sample)
+
+            poly = burst.border[0]
+            # Give some margin to the polygon
+            margin = 0.001
+            poly = poly.buffer(margin)
+            burst_map['border'].append(Polygon(poly.exterior.coords).wkt)
+
+            # Transform coordinates from lat/long to EPSG
+            llh = osr.SpatialReference()
+            llh.ImportFromEPSG(4326)
+            tgt = osr.SpatialReference()
+
+            tgt_x, tgt_y = [], []
+            x, y = poly.exterior.coords.xy
+            tgt.ImportFromEPSG(int(epsg))
+            trans = osr.CoordinateTransformation(llh, tgt)
+            for lx, ly in zip(x, y):
+                dummy_y, dummy_x, dummy_z = trans.TransformPoint(ly, lx, 0)
+                tgt_x.append(dummy_x)
+                tgt_y.append(dummy_y)
+            
+            if int(epsg)==4326:
+              x_min = x_spacing * (min(tgt_x) / x_spacing)
+              y_min = y_spacing * (min(tgt_y) / y_spacing)
+              x_max = x_spacing * (max(tgt_x) / x_spacing)
+              y_max = y_spacing * (max(tgt_y) / y_spacing)
+            else:
+              x_min = x_spacing * round(min(tgt_x) / x_spacing)
+              y_min = y_spacing * round(min(tgt_y) / y_spacing)
+              x_max = x_spacing * round(max(tgt_x) / x_spacing)
+              y_max = y_spacing * round(max(tgt_y) / y_spacing) 
+
+            # Allocate coordinates inside the dictionary
+            burst_map['min_x'].append(x_min)
+            burst_map['min_y'].append(y_min)
+            burst_map['max_x'].append(x_max)
+            burst_map['max_y'].append(y_max)
+
+    # Save generated burst map as csv
+    data = pd.DataFrame.from_dict(burst_map)
+    data.to_csv(output_filename)
+
+    # Create GeoDataFrame to plot bursts on a map
+    df = data
+    df['border'] = df['border'].apply(wkt.loads)
+    gdf = gpd.GeoDataFrame(df, crs='epsg:4326')
+    gdf2 = gdf.rename(columns={'border': 'geometry'}).set_geometry('geometry')
+    
+    # Save the GeoDataFrame as a shapefile (some people may prefer the format)
+    gdf2.to_file('gdf2.shp')
```

Comment:
```suggestion
    gdf2.to_file(f'{output_filename}.shp')
```
Something like this to make all the outputs per run distinct?

Maybe change `output_filename` to `output_file_prefix` and hard code file extension accordingly in the f-string in write/output?

Reply:
Good idea and then you could use the same for kml, html outputs